addons/website/static/src/less/website.snippets.less

```diff
@@ -53,9 +53,32 @@ table.table_desc tr td {
 }
 
 // Other snippet styles
+.s_feature_grid p{
+    overflow: hidden;
+}
```

Comment:
To prevent text coming below the icon / image like this : 
![feature-grid-overflow](https://cloud.githubusercontent.com/assets/7599908/26618102/7e38336e-45f6-11e7-9750-c9336acfd0f9.png)